File: Level1/최대공약수와 최소공배수.py
#최대공약수와 최소공배수
def solution(n, m):
    def gcd(n,m):
        if n%m==0:
            return m
        else:
            return gcd(m,n%m)
    answer = []
    answer.append(gcd(n,m)) ##최대공약수
    answer.append(n*m//gcd(n,m)) ##최소공배수
    return answer
n,m=map(int,input().split())
print(solution(n,m))

File: Level1/최대공약수와 최소공배수.cpp
#include <string>
#include <vector>

using namespace std;

int gcd(int a, int b) {
    if (a % b == 0)
        return b;
    else
        return gcd(b, a % b);
}

vector<int> solution(int n, int m) {
    vector<int> answer;
    
    int gcd_value = gcd(n, m);
    int lcm_value = (n * m) / gcd(n, m);
    answer.push_back(gcd_value);
    answer.push_back(lcm_value);
    return answer;
}